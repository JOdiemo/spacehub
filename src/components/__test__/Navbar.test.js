import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../NavBar';

describe('Navbar component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Router><Navbar /></Router>);
    expect(tree).toMatchSnapshot();
  });
});