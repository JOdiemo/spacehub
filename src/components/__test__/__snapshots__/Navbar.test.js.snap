// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component matches the snapshot 1`] = `
<nav
  className="nav-bar"
>
  <img
    alt="Logo"
    className="logo"
    src="space.png"
  />
  <h1
    className="main-heading"
  >
    Space Hub
  </h1>
  <ul
    className="navlinks"
  >
    <li>
      <a
        activeClassName="active-link"
        aria-current="page"
        className="link active"
        exact={true}
        href="/"
        onClick={[Function]}
      >
        Rockets
      </a>
    </li>
    <li>
      <a
        activeClassName="active-link"
        className="link"
        exact={true}
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li>
      <a
        activeClassName="active-link"
        className="link"
        exact={true}
        href="/myprofile"
        onClick={[Function]}
      >
        My Profile
      </a>
    </li>
  </ul>
</nav>
`;
